### DISCUSSION TITLE
Assigned adresses not showing in "IP Address" column

### DISCUSSION THREAD

**Discussion by IIlyasha** [netbox-community/netbox#12676] [Help Wanted!]
I have a server with 2 ip addresses asigned to different ports but only primary address showing in "IP Adresses". Its a bug or i do something wrong?

**Comment by a084ed22** [comment]
The interface allows to map only a single address as primary for the server/device/VM - more specifically, one IPv4 and one IPv6.

**Reply by IIlyasha** [reply]
Its means IP Address column its summaray of ipv4 and ipv6 primary addresses?
![Screenshot from 2023-05-22 13-05-57](https://github.com/netbox-community/netbox/assets/90531596/e323ffd9-0455-4fcc-9bcd-b5d38252635f)

**Comment by mtinberg** [comment]
Device/VM ip_address/ip4_address/ip6_address are for the single primary IP that should be used to manage the device, per-interface IP addresses are listed on the Interfaces tab.  There isn't a Device/VM association summary for all addresses on all interfaces


—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________
